### PR TITLE
Various minor text string fixes/adjustments

### PR DESCRIPTION
**Pull request by zyx-xyzzy**
A whole bunch of grammar fixes, along with a handful of miscellaneous string improvements like making some of the helptext more descriptive. Hopefully I caught all the points where the renamed towns/dungeons are referenced; lmk if there are any I missed that still need to be switched over.

### CODE REVIEW

File: src/components/FAQModal.html [hunk 1]
```diff
@@ -61,12 +61,12 @@ <h5>Frequently Asked Questions</h5>
             <span class="text-muted">There is a maximum of 3 active alots.</span></p>
             <p><h5 class="text-primary"><strong>Gameplay:</strong></h5></p>
             <p><strong>How many regions are playable?</strong><br/>
-            <span class="text-muted">Currently Kanto through to Kalos are able to be played.</span></p>
+            <span class="text-muted">Currently Kanto through to Alola are able to be played.</span></p>
```
Comment (Aegyo) [inline src/components/FAQModal.html:64]:
```suggestion
            <span class="text-muted">Currently Kanto through to <span data-bind="text: GameConstants.camelCaseToString(GameConstants.Region[GameConstants.MAX_AVAILABLE_REGION])">...</span> are able to be played.</span></p>
```

File: src/components/underground.html [hunk 2]
```diff
@@ -148,11 +148,11 @@ <h4><u>Treasures</u></h4>
 
                     <h4><u>Upgrades</u></h4>
                     <p>You can use your diamonds to buy all kinds of upgrades that will help you explore the underground even further!</p>
-                    <p>Some upgrades will make the Underground harder and are less cost efficient.
+                    <p>Some upgrades will make the Underground harder and are less cost-efficient.
                     <br/>Avoid these until you have the other available upgrades!</p>
 
                     <h4><u>Daily deals</u></h4>
-                    <p>In the Daily deals tab you can trade your items.</p>
+                    <p>In the Daily deals tab you can trade your items. The trades available rotate each day at midnight.</p>
```
Comment (Aegyo) [inline src/components/underground.html:155]:
```suggestion
                    <p>In the Daily deals tab you can trade your items. The trades available change each day at midnight.</p>
```
`rotate`, to me, implies there is some fixed rotation where today's deals will eventually roll back around, but that is not the case.

**Review by Aegyo — COMMENTED** [review]
I haven't tested the town + dungeon name changes, but the rest looks good :+1:

File: src/scripts/farming/Farming.ts [hunk 3]
```diff
@@ -384,7 +384,7 @@ class Farming implements Feature {
         this.berryData[BerryType.Maranga]   = new Berry(BerryType.Maranga,  [5040, 10080, 20160, 40320, 80640],
             1, 0.05, 3100, 20,
             [10, 10, 30, 30, 10], BerryColor.Blue,
-            ['This Berry resembles the Durin Berry, however its spikes are less pronounced. It is quite delicious when roasted.'], undefined, ['Flabébé (Blue)']);
+            ['This Berry resembles the Durin Berry; however, its spikes are less pronounced. It is quite delicious when roasted.'], undefined, ['Flabébé (Blue)']);
```
[the remark this answers is not in the document]
Comment (zyx-xyzzy) [reply src/scripts/farming/Farming.ts:387]: It's more technically correct than the old version, but now that you mention it the construction is a bit clunky there, yeah. Would something like one of these work better, maybe?

"This Berry resembles the Durin Berry, but its spikes are less pronounced."
"This Berry resembles the Durin Berry, though its spikes are less pronounced."

[the remark this answers is not in the document]
Comment (Aegyo) [reply src/scripts/farming/Farming.ts:387]: The "though" variant sounds best to me :+1:

[the remark this answers is not in the document]
Comment (zyx-xyzzy) [reply src/scripts/farming/Farming.ts]: Alright, done! 👍

**Review by Aegyo — DISMISSED** [review]
Got around to checking all the locations, I think it just needs the image names in `src/assets/towns` changing :)

**Pull request merged**